docs/userGuide/templates.md

```diff
@@ -20,5 +20,5 @@ markbind init --template minimal
 
 Name    | Template key | Description | Quick Deploy
 ----    | -------      | ----------- | ------------
-Default | `default`    | Default template if `--template` is unspecified. Filled with MarkBind features to guide you to author better content. | <a href="https://app.netlify.com/start/deploy?repository=https://github.com/MarkBind/init-typical-netlify"><img src="https://www.netlify.com/img/deploy/button.svg" /></a>
-Minimal | `minimal` | Minimalistic template that gets you started quickly | <a href="https://app.netlify.com/start/deploy?repository=https://github.com/MarkBind/init-minimal-netlify"><img src="https://www.netlify.com/img/deploy/button.svg" /></a>
\ No newline at end of file
+Default | `default`    | Default template if `--template` is unspecified. Includes links to the main sections of the MarkBind User Guide. | <a href="https://app.netlify.com/start/deploy?repository=https://github.com/MarkBind/init-typical-netlify"><img src="https://www.netlify.com/img/deploy/button.svg" /></a>
```

Comment:
I think we might not need to mention that there are links to the UG here, since it's not really a feature of the template. Maybe we can relate this to the template being a quick start option?

Reply:
Good point! Would something like "Includes core features such as site and page navigation for a more convenient quick start." be better?